client-participation/js/strings/sk.js

```diff
@@ -0,0 +1,124 @@
+s.helpWriteListIntro = "Čo tvorí dobrý návrh?";
+s.helpWriteListStandalone = "Samostatná myšlienka";
+s.helpWriteListRaisNew = "Originálna perspektíva, či skúsenosť";
+s.helpWriteListShort = "Zreteľný obsah a stručný opis (maximálne 140 znakov)";
+s.heresHowGroupVoted = "Takto hlasovala skupina {{GROUP_NUMBER}}:";
+s.one_person = "{{x}} osoba";
+s.x_people = "{{x}} osoby";
+s.acrossAllPtpts = "Naprieč všetkými účastníkmi:";
+s.xPtptsSawThisComment = " videlo tento výrok";
+s.xOfThoseAgreed = "z týchto účastníkov súhlasilo";
+s.xOfthoseDisagreed = "z týchto účastníkov nesúhlasilo";
+s.opinionGroups = "Názorové skupiny";
+s.topComments = "Top výroky";
+s.divisiveComments = "Polarizujúce výroky";
+s.pctAgreed = "{{pct}}% Súhlasilo";
+s.pctDisagreed = "{{pct}}% Nesúhlasilo";
+s.pctAgreedLong = "{{pct}}% zo všetkých hlasujúcich o tomto výroku {{comment_id}} súhlasilo.";
+s.pctAgreedOfGroup = "{{pct}}% zo skupiny {{group}} súhlasilo";
+s.pctDisagreedOfGroup = "{{pct}}% zo skupiny {{group}} nesúhlasilo";
+s.pctDisagreedLong = "{{pct}}% zo všetkých hlasujúcich o tomto výroku {{comment_id}} nesúhlasilo.";
+s.pctAgreedOfGroupLong = "{{pct}}% zo skupiny {{group}}, ktorí hlasovali za výrok {{comment_id}} súhlasilo.";
+s.pctDisagreedOfGroupLong = "{{pct}}% zo skupiny {{group}}, ktorí hlasovali za výrok {{comment_id}} nesúhlasilo.";
+s.commentSent = "Výrok úspešne pridaný! Váš názor a to ako ste hlasovali uvidia len ostatní účastníci.";
+s.commentSendFailed = "Chyba pri zadaní výroku.";
+s.commentSendFailedEmpty = "Chyba pri zadaní výroku - Políčko pre výrok nemôže byť prázdne.";
+s.commentSendFailedTooLong = "Chyba pri zadaní výroku - Výrok príliš dlhý.";
+s.commentSendFailedDuplicate = "Chyba pri zadaní výroku - Už existuje identický výrok.";
+s.commentErrorDuplicate = "Duplikát! Tento výrok už existuje.";
+s.commentErrorConversationClosed = "Táto diskusia už bola uzatvorená. Ďalšie názory už nemôžu byť pridané.";
+s.commentIsEmpty = "Výrok je prázdny";
+s.commentIsTooLong = "Výrok je pridlhý";
+s.hereIsNextStatement = "Váš hlas bol úspešne zaznamenaný. Prejdite vyššie pre zobrazenie ďalšieho výroku.";
+
+s.connectFacebook = "Pripoj Facebook";
+s.connectTwitter = "Pripoj Twitter";
+s.connectToPostPrompt = "Pridaj svoju identitu pre zaznamenanie výroku. Na vašu časovú os nebudeme pridávať žiadne príspevky.";
+s.connectToVotePrompt = "Pridaj svoju identitu pre hlasovanie. Na vašu časovú os nebudeme pridávať žiadne príspevky.";
+s.tip = "Tip:";
+s.commentWritingTipsHintsHeader = "Tipy pre písanie výrokov";
+s.tipCharLimit = "Výroky sú limitované na {{char_limit}} znakov.";
+s.tipCommentsRandom = "Návrhy sú zobrazované náhodne a neodpovedáte priamo na výroky ostatných účastníkov.";
+s.tipOneIdea = "Rozložte dlhé výroky obsahujúce viacero myšlienok. Ostatným úšatníkom sa tak bude ľahšie hlasovať o vašom výroku.";
+s.tipNoQuestions = 'Výroky by nemali byť v podobe otázok. Účastníci budú hlasovať "súhlasím" alebo "nesúhlasím" o vašich výrokoch.';
```

Comment:
@dvagala Small question: I don't think we use quotes in any other language. Are they required to capture the same meaning? Or might this be a sign there are other, more natural agree/disagree words that are used in more casual speaking? Is conjugation in the first-person common in Slovakian user interfaces?

The agree/disagree/pass words are usually the ones we try hardest to capture same meaning, so I hope it doesn't seem annoying for me to ask :)

Reply:
Yup, the first-person is common in Slovak UI. It's not a rule, but in this case it sounds more natural :)
This is exactly the reason why the surrounding quotes. Because the words are not in the normal form, it would be difficult to put it in that sentence while keeping the correct meaning. Hope it makes sense :D

Reply:
Totally. Thanks for explaining! I'll leave this open in case the Polis Team is also curious, but I'm satisfied. (I'm just a community contributor, and will leave it to a member of the Polis Team to merge this)

Reply:
Okay, thanks :)